environment.yml

```diff
@@ -2,12 +2,46 @@
 # conda env create -f environment.yml
 name: open-gira
 channels:
-  - defaults
   - conda-forge
+  - bioconda
+  - defaults
 dependencies:
   - python=3.9
   - pip
-  - pip:  # delegate to pip for majority of packages
-    - -r requirements.txt
+  - pip:  # delegate to pip for non-conda packages
+    - nismod-snail==0.2.1 # vector-raster intersections
+    - snkit==1.7.1        # spatial network cleaning
+    - osmium==3.2.0       # OpenStreetMap protobuf handling
+  - black               # formatter
+  - cython==0.29.28     # c--python interface
   - datashader          # plotting large datasets
+  - flake8              # linter
+  - geopandas>=0.11.0   # geospatial dataframes
+  - geopy               # geocoding client
+  - ipykernel           # notebook support
+  - jupyter             # notebook support
+  - matplotlib==3.5.1   # basic plotting
+  - nb_conda_kernels    # notebook support
+  - netCDF4             # NetCDF file format
+  - networkx            # networks
+  - numpy~=1.22.0       # data arrays
+  - openpyxl            # Excel file format
+  - pathos              # multiprocessing
+  - pyarrow             # parquet file format, Arrow data exchange
+  - aws-sdk-cpp=1.8.186
+  # workaround for a bug in aws-sdk-cpp, which is used by pyarrow
+  # https://issues.apache.org/jira/browse/ARROW-15141
+  # https://github.com/apache/arrow/pull/14157
+  - pyproj>=3.3.1       # geographic projections
+  - pyrosm              # python OpenStreetMap network reader
+  - pytest==6.2.4       # testing
+  - pytest-xdist        # parallelise testing across CPUs
+  - rasterio            # raster file formats
+  - rasterstats         # raster statistics
+  - requests            # make HTTP requests
   - spatialpandas       # plotting large datasets
+  - snakemake==7.15.1   # workflow management
```

Comment:
Should `snakemake` be here in the end? Docs above mention that it's not included, but it was added back as of a70347284e22a2d7fb6425d0af16349954d1ae90

Reply:
After that problem which I thought was conflicting installs, it's now working on ARC with snakemake included in the environment file. I have no idea what I did.

It is neater to leave it out, then when executing on ARC we don't install a pointless package. However it's simpler to setup locally with snakemake included in the environment file. Do you have any preference either way?

Reply:
Have included it and updated docs to reflect that